app/components/calendar.jsx

```diff
@@ -73,12 +198,15 @@ const MyCalendar = () => {
                                 onClick={handleEventAdd}
                                 className="bg-blue-500 text-white px-4 py-2 rounded"
                             >
-                                Add Event
+                                {isEditing ? 'Update Event' : 'Add Event'}
                             </button>
                         </div>
                     </div>
                 </div>
             )}
+
+            {/* Toast Notifications */}
+            <ToastContainer position="top-right" autoClose={5000} closeOnClick />
         </div>
     );
 };
```

Comment:
The code works well for managing events and reminders but has duplicate imports and some repeated UI elements. Refactoring can improve clarity and reduce redundancy.